tests/api/test_ars.py

```diff
@@ -0,0 +1,26 @@
+from pprint import pprint
+
+
+class TestSaiArs:
```

Comment:
Please do not spread the same changes across multiple PRs. It make the review process inefficient. E.g., these TCs are also in https://github.com/opencomputeproject/SAI-Challenger/pull/196/files and https://github.com/opencomputeproject/SAI-Challenger/pull/202/files . Please check opened PRs and clean them up. Thanks.

Reply:
@andriy-kokhan  do you mean to remove pprint import pprint line from the scripts?

Reply:
@andriy-kokhan do you mean to remove pprint import pprint line from the scripts?

Reply:
No. I mean `TestSaiArs` class with related TCs is defined in all three PRs. The same for a few other classes. So, what should be reviewed? That's confusing..